RA Scripts/Legend of Zelda, The The Minish Cap.rascript

```diff
@@ -596,19 +597,26 @@ function GenerateSaveProtectedAchievementFromConditions(cheevo, conditions)
     )
 }
 
+function MusicId() => byte(0x029EF4)
+
+function CreditsLoaded() => MusicId() == 0x09 && prev(MusicId()) == 0x00
+
+function ScreenIdByte() => byte(0x000BF4)
+function ScreenIdWord() => word(0x000BF4)
```

Comment:
The other cheevos in the past used different measurements for the same memory address. I would have to go through the game again in more depth to make it consistent with word throughout.

Reply:
In that case, you could probably just use the byte version in the places you touched and remove the word version. There's really no need to have both. And after removing it, eliminating the width marker from the name would also help with code clarity.

Reply:
Except the address itself is a 16-bit address. Possibly 16-bit Big Endian from what I noticed as I was taking notes. I'd think word would be the more accurate value to keep. Furthermore, some of the cheevos require word and not byte (look at Bested Mazaal, Gambler, and Bested Big Blue Chuchu for examples).

Reply:
Big endian!? Yikes, that really would require additional testing. I guess this is a necessary evil, then. Adding a todo above these functions to merge them later will suffice, I think. Besides that, I'll accept this change so that you don't have to be subject to replaying the whole game again just to test this.